src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumManageBoardsPageObject.java

```diff
@@ -138,8 +138,8 @@ public void verifyForumNotExists(String forumName, String wikiURL) {
     } catch (UnsupportedEncodingException e) {
       PageObjectLogging.log("verifyForumNotExists", e, false);
     }
-    wait.forElementVisible(By.xpath(
-        "//div[contains(text(), \"We couldn't find a board with that title.  Here's the list of forum boards.\")]"));
+//    wait.forElementVisible(By.xpath(
+//        "//div[contains(text(), \"We couldn't find a board with that title.  Here's the list of forum boards.\")]"));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

Reply:
Łejery tej!